py/DREAM/Settings/Equations/RunawayElectrons.py

```diff
@@ -229,6 +254,12 @@ def fromdict(self, data):
                 self.comptonPhotonFlux  = data['compton']['flux']
                 self.comptonPhotonFlux_t = np.array([0.0])
 
+        if 'C1' in data['compton']:
+            self.C1_Compton = data['compton']['C1']
+            self.C2_Compton = data['compton']['C2']
+            self.C3_Compton = data['compton']['C3']
```

Comment:
Since you're checking explicitly for ``float`` in the ``verify()`` method, you should explicitly typecast these inputs to floats (otherwise they may be e.g. ``numpy.float64``, which will not be tolerated by ``verify()``).